src/core/qgsthreadingutils.h

```diff
@@ -23,8 +23,10 @@
 
 #include "qgsfeedback.h"
 
+#ifndef SIP_RUN
```

Comment:
Why do we need this if we have already SIP_NO_FILE ?

Reply:
because sipify.py still parsed the file and crashed due to the '\' at the end of lines. I remove the `#define SIP_NO_FILE`

Reply:
CI crashed ... both are needed!

Reply:
I tried but I am not proud of what I have done. Can you check please?

Reply:
I don't see any modifications to sip_include.sh and still see `#ifndef SIP_RUN`

Reply:
my bad, it was in qgslogger